src/main/java/org/libsdl/app/SDLActivity.kt

```diff
@@ -104,12 +104,13 @@ open class SDLActivity internal constructor (context: Context?) : RelativeLayout
         val zeroRect = RectF(0f, 0f, 0f, 0f)
 
         if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.R) {
-            val insets = rootView.rootWindowInsets?.getInsets(
-                WindowInsets.Type.displayCutout() or
-                WindowInsets.Type.navigationBars()
-            ) ?: return zeroRect
+            val activity = context as Activity
+            val typeMask =
+                if (activity.window.navigationBarColor == Color.TRANSPARENT) WindowInsets.Type.displayCutout() or WindowInsets.Type.navigationBars()
```

Comment:
this is a hack, it's not only the color which has to be set to make content appear behind the system navigation, but I found this to be the most reliable way to.

Reply:
In our case we ask for navigation insets for devices with gesture navigation enabled